### DISCUSSION TITLE
Multiple windows and sections

### DISCUSSION THREAD

**Discussion by curiosport** [sumatrapdfreader/sumatrapdf#2304] [Q&A]
Currently I am using the `-appdata` command for the sections but when I add `-new-window` nothing happens.

My intention is to be able to open each section in its own window without having to make multiple copies of Sumatra.exe

How can I do this?

**Comment by kjk** [comment]
`-new-window` is only applied when you also provide a name of the file to open i.e. `SumatraPDF.exe -new-window foo.pdf`

I'm guessing you're just doing `SumatraPDF.exe -new-window -appdata <dir>` and I didn't anticipate such usage.

Please open a [new issue](https://github.com/sumatrapdfreader/sumatrapdf/issues) if you want `-new-window` to always create a new window, even if file is not provided.
